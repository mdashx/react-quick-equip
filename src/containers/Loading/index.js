import React from 'react';

class Loading extends React.Component {
  render() {
    return (
      <h2>Loading...</h2>
    );
  }
}

export default Loading;
