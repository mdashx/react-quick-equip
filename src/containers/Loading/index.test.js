import React from 'react';
import renderer from 'react-test-renderer';
import Loading from 'containers/Loading';

test('Loading shows message in h2 tag', () => {
  const component = renderer.create(<Loading />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

