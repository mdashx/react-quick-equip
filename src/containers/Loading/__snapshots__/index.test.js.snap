// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading shows message in h2 tag 1`] = `
<h2>
  Loading...
</h2>
`;
